### PR TITLE
Lesson 6 in spanish

### PR DESCRIPTION
**Pull request by zwoosks**
- [x] I did these translations myself and own copyright for them
- [x] I didn't use a machine to do these translations
- [x] I assign all copyright to Loom Network for these translations

### CODE REVIEW

**Comment by JuanitoH29** [comment]
Aun sigue faltando partes por traducir, son todos los // 😅

File: es/6/01.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+// Usando Bower
+bower install web3
+// ...etc.
+```
+
+O simplemente puedes descargar el minificado `.js` archivo desde <a href="https://github.com/ethereum/web3.js/blob/1.0/dist/web3.min.js" target=_blank>github</a> y después incluirlo en tu proyecto:
+```
+<script language="javascript" type="text/javascript" src="web3.min.js"></script>
+```
+Ya que no queremos hacer demasiadas suposiciones sobre su entorno de desarrollo y qué administrador de paquetes va a utilizar, en este tutorial simplemente vamos a incluir Web3 en nuestro proyecto usando un script como el ejemplo de arriba.
```
Comment (jcaracciolo) [inline es/6/01.md]: *su* por *tu* para mantener consistencia (lo mismo con *va* y *vas*)

File: es/6/01.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+## Póngalo a prueba
```
Comment (jcaracciolo) [inline es/6/01.md]: *Pongalo por *Ponlo* para mantener consistencia

**Comment by eduardonunesp** [comment]
@zwoosks thanks for your work, I think the files `index.json` and `share-templates-msgs.json` should be translated too

File: es/6/index.json [hunk 3]
```diff
@@ -52,254 +52,219 @@
...
+                "¡Enseña el ejército zombi a tus amigos!",
+                "Comparte esta URL para que tus amigos puedan ver tu ejército:"
             ],
             "footer": [
-                "La Leçon 6 sera disponible dans les prochaines semaines !",
-                "Nous vous enverrons un email pour vous tenir au courant.",
-                "En attendant, rejoignez-nous sur {telegramLink}, ou suivez-nous sur {twitterLink} pour vous joindre à la conversation !"
+                "¿Estás drisfrutando CryptoZombies? ¡Háznoslo saber!",
+                "Únete a la conversación en {telegramLink}, o síguenos en {twitterLink}",
+                "Cuando estés preparado, clica en el botón de abajo para acceder a la Lección 6:
```
Comment (eduardonunesp) [inline es/6/index.json]: Looks like the JSON file is breaking here

File: es/6/index.json [hunk 4]
```diff
@@ -52,254 +52,219 @@
...
             ],
```
Comment (eduardonunesp) [inline es/6/index.json]: Looks like the JSON file is breaking here

File: es/6/index.json [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+                "Funciones de pago, y como ganar dinero con tus juegos basados en Ethereum",
+                "Retirar ETH de los contratos inteligentes",
+                "Generar numeros aleatorios y la seguridad en Ethereum",
+                "¡Y más!"
+            ],
+            "callToAction": [
+                "¿Quiéres crear tu propio CryptoZombie y añadirlo a tus filas?",
+                "¡Aprende como crear tus propios juegos en Ethereum GRATIS con CryptoZombies! Empieza ahora:"
+            ]
+        }
```
Comment (eduardonunesp) [inline es/6/index.json]: Thanks for the previous fix, here we also need JSON data for lesson 5 and 6 :)

Comment (zwoosks) [reply es/6/index.json]: Excuse me jejejej. Now it's done.

**Comment by eduardonunesp** [comment]
Thanks @zwoosks

**Pull request merged**